devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/picocli-example/java/src/main/java/org/acme/picocli/HelloCommand.java

```diff
@@ -0,0 +1,38 @@
+package org.acme.picocli;
+
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+import java.io.File;
+import java.net.URL;
+
+@Command(name = "hello")
+public class HelloCommand implements Runnable{
+
+    @CommandLine.ArgGroup(exclusive = true, multiplicity = "1")
+    Name name;
+    static class Name {
+
+
+        @CommandLine.Option(names = {"--first-name"}, description = "The guest first name")
```

Comment:
this felt a bit weird to have them exclusive, maybe `--firt-name` and `--nick-name`? would make more sense?

Reply:
Ok yes of course! I only intended to show the exclusive feature, I will make the change.